bindgen/src/bindings/cpp/templates/wrapper.cpp

```diff
@@ -9,6 +9,29 @@ namespace {{ namespace }} {
 namespace uniffi {
 template <class> inline constexpr bool always_false_v = false;
 
+static bool initialized = false;
```

Comment:
I wonder how this would work with multiple uniffi bindings in the same process. Would multiple shared libraries shared the same `initialized` variable memory?

Reply:
Actually, I guess it would not be multiple shared libraries, but rather multiple bindings compiled into the same library, which doesn't work right now anyway.